src/vm/reflectioninvocation.cpp

```diff
@@ -1431,8 +1431,18 @@ FCIMPL4(Object*, RuntimeMethodHandle::InvokeMethod,
         }
 #endif
 
+#ifdef _TARGET_ARM64_
```

Comment:
No, I was suggesting that to keep the code uniform for all platforms. All platforms now have ArgIterator::GetArgLocDescForStructInRegs and it just always returns NULL on non-amd64. So it seems strange to deviate from that for ARM64.